cordova-plugin-appcenter-crashes/src/android/plugin.gradle

```diff
@@ -0,0 +1,9 @@
+repositories {
```

Comment:
This file needs to be removed before releasing as this would inject this repo into everyone's applications. This needs to be in the release procedure.

Reply:
addressed.